vocode/streaming/synthesizer/azure_synthesizer.py

```diff
@@ -221,6 +221,18 @@ async def create_speech(
         offset = 0
         self.logger.debug(f"Synthesizing message: {message}")
 
+        async def empty_generator():
```

Comment:
tiny optimization: let's put this outside of the function since it's essentially a constant

Reply:
Done :+1: